drivers/github-releases-api.go

```diff
@@ -0,0 +1,66 @@
+package drivers
+
+import (
+	"errors"
+)
+
+// GitHubReleasesAPI is a driver, that obtains release information
+// directly from GitHub releases API https://developer.github.com/v3/repos/releases/
+//
+// For each obtained release callback function will be invoked
+// Execution will be stopped when callback returns null or there are no more releases
+//
+// Flow:
+// 1. Driver reads latest release and sends it to callback
+//    https://developer.github.com/v3/repos/releases/#get-the-latest-release
+//    Example: https://api.github.com/repos/facebook/react/releases/latest
+// 2. If there were no errors it fetches releases list
+//    https://developer.github.com/v3/repos/releases/#list-releases-for-a-repository
+// 3. TODO If there is still no error - it traverses across other pages
+//
+//
+// PS. TODO implement authentication
+func GitHubReleasesAPI(repository string, callback func(title, body string) error) error {
```

Comment:
I think in future we need ability to handle "3xx: redirect" status codes. I propose add this thing in todo list